cpu/stm32/periph/i2c_2.c

```diff
@@ -125,11 +127,56 @@ static void _init_pins(i2c_t dev)
     gpio_init_af(i2c_config[dev].scl_pin, GPIO_AF_OUT_OD);
     gpio_init_af(i2c_config[dev].sda_pin, GPIO_AF_OUT_OD);
 #else
+    gpio_init(i2c_config[dev].scl_pin, GPIO_OD_PU);
+    gpio_init(i2c_config[dev].sda_pin, GPIO_OD_PU);
     gpio_init_af(i2c_config[dev].scl_pin, i2c_config[dev].scl_af);
     gpio_init_af(i2c_config[dev].sda_pin, i2c_config[dev].sda_af);
 #endif
 }
 
+static void _deinit_pins(i2c_t dev)
+{
+    gpio_set(i2c_config[dev].scl_pin);
+    gpio_set(i2c_config[dev].sda_pin);
+    gpio_init(i2c_config[dev].scl_pin, GPIO_OD);
+    gpio_init(i2c_config[dev].sda_pin, GPIO_OD);
```

Comment:
Please add a comment here why this setting was chosen

Reply:
I added the comment